CHANGES.txt

```diff
@@ -1,4 +1,6 @@
 4.0-beta3
+ * Fix a race condition on ColumnFamilyStore and TableMetrics (CASSANDRA-16228)
+ * Make CacheMetrics.requests of type meter to skip the need of customizations of codadhale functions (CASSANDRA-16228)
```

Comment:
The true problem behind the `Metered` approach is that the metrics reporter looks for metrics classes that are instance of the standard codahale classes. Due to that, other `Metered` implementation will not be exposed through the reporter.
We should use something like: `Ensure that CacheMetrics.requests is picked up by the metric reporter`